guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -288,7 +290,7 @@ namespace.readNumber&nbsp;=&nbsp;function(x)&nbsp;{
 </pre>
 
 
-<h3 id="frontend-dev-guide-javascript-js-coding-docblock-jstypes">JavaScript types</h3>
+### JavaScript types
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-jstypes}` to preserve the custom id.